tests/integration/upgrade_test.go

```diff
@@ -173,6 +216,22 @@ func (s *UpgradeSuite) updateClusterRoles() error {
 	}
 
 	newResources := `
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Does this need to be added to `upgrade-from-v0.9-create.yaml`?

Reply:
@BlaineEXE Yes, good catch, thanks